requirements.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install the list of required packages
+  package: name={{ item }} state=installed
+  with_items:
+  - git
+  - rpm-ostree
+  - libvirt
+  - qemu-img
+
+- name: Download daemon rpm
+  get_url:
+    url: http://libslack.org/daemon/download/daemon-0.6.4-1.x86_64.rpm
+    dest: /tmp
+
+- name: Install daemon
+  command:
+    rpm --install /tmp/daemon-0.6.4-1.x86_64.rpm
+  ignore_errors: yes
```

Comment:
I'm using `ignore_errors` so that the playbook won't fail if `daemon` is already installed.